bin/main.rb

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+require_relative '../lib/methods.rb'
+
+class UserInterface
+  def initialize
+    @game = Game.new
```

Comment:
- `Game` here is supposedly the **Methods** class that you have in the `lib` folder